kafka-connect-s3/src/test/java/io/confluent/connect/s3/storage/ElasticByteBufferTest.java

```diff
@@ -0,0 +1,327 @@
+package io.confluent.connect.s3.storage;
+
+import org.apache.commons.lang.RandomStringUtils;
+import org.apache.commons.lang.math.RandomUtils;
+import org.junit.Test;
+import java.nio.BufferOverflowException;
+import static org.junit.Assert.*;
```

Comment:
super small nit- .* import 
```suggestion
import static org.junit.Assert.assertEquals;
import static org.junit.Assert.assertFalse;
import static org.junit.Assert.assertNotSame;
import static org.junit.Assert.assertSame;
import static org.junit.Assert.assertThrows;
import static org.junit.Assert.assertTrue;
```